cmd/hub-router/go.mod

```diff
@@ -11,17 +11,18 @@ require (
 	github.com/cenkalti/backoff/v4 v4.1.0
 	github.com/containerd/continuity v0.0.0-20200928162600-f2cc35102c2a // indirect
 	github.com/gorilla/mux v1.7.4
-	github.com/hyperledger/aries-framework-go v0.1.6-0.20210305161732-29081cf55c77
-	github.com/hyperledger/aries-framework-go-ext/component/storage/mysql v0.0.0-20210305233053-d3d22c802c12
-	github.com/hyperledger/aries-framework-go/component/storageutil v0.0.0-20210224230531-58e1368e5661
-	github.com/hyperledger/aries-framework-go/spi v0.0.0-20210302153503-0e00e248f14d
+	github.com/hyperledger/aries-framework-go v0.1.6
+	github.com/hyperledger/aries-framework-go-ext/component/storage/mysql v0.0.0-20210306170115-156a24580a5c
+	github.com/hyperledger/aries-framework-go/component/storageutil v0.0.0-20210306162754-1a1e0c4a378e
+	github.com/hyperledger/aries-framework-go/spi v0.0.0-20210306162754-1a1e0c4a378e
```

Comment:
shouldn't all these be release revisions?

Reply:
There is no release version for sub-modules yet - ref - https://github.com/trustbloc/hub-router/pull/64